eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/EventReceiverTest.java

```diff
@@ -0,0 +1,375 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I have no problems splitting the EventReceiverTest integration tests into multiple classes if it makes reasoning about these chained scenarios easier.